internal/repository/verify.go

```diff
@@ -43,6 +44,26 @@ func (r *Repository) VerifyRef(ctx context.Context, target string, latestOnly bo
 	return r.verifyRefTip(target, expectedTip)
 }
 
+func (r *Repository) VerifyRefFromEntry(ctx context.Context, target, entryID string) error {
+	if !dev.InDevMode() {
```

Comment:
If this is a dev mode feature, we should try to make this more obvious from the flag options. e.g. we should move a variant of this command to `gittuf dev` (i.e. `gittuf dev verify-ref`), mark the flags as hidden, or worst case make sure the flag is documented.

Reply:
Good catch, I'm going to add the dev mode only message to the flag.

Reply:
Done!